packages/grafana-ui/src/components/PanelChrome/PanelContext.ts

```diff
@@ -22,6 +22,11 @@ export interface PanelContext {
   onAnnotationCreate?: (annotation: AnnotationEventUIModel) => void;
   onAnnotationUpdate?: (annotation: AnnotationEventUIModel) => void;
   onAnnotationDelete?: (id: string) => void;
+  /**
+   * splitOpenFn is used in Explore to open the split view. It can be used in panels which has intercations and used in Explore as well.
+   * For example TimeSeries panel.
+   */
+  splitOpenFn?: SplitOpen;
```

Comment:
```suggestion
  onSplitOpen?: SplitOpenFn;
```

Reply:
would it make sense if I'd only rename the property and not the type?

Reply:
yes